numalogic/config/_config.py

```diff
@@ -83,7 +83,9 @@ class TrainerConf:
 class NumalogicConf:
     """Top level config schema for numalogic."""
 
-    model: ModelInfo = field(default_factory=ModelInfo)
+    model: ModelInfo = field(
+        default_factory=lambda: ModelInfo(name="VanillaAE", conf={"n_features": 2, "seq_len": 10})
+    )
```

Comment:
Any reason why are we doing this? We should not have a default for the number of features arg

Reply:
I was keeping everything to have a default. Wanted to get your thoughts on this too.
Can revert it back if needed